src/pages/images/RemoteImageSelector.tsx

```diff
@@ -212,7 +229,7 @@ const ImageSelector: FC<Props> = ({ onClose, onSelect }) => {
         ],
         sortData: {
           os: item.os.toLowerCase(),
-          release: item.release.toLowerCase(),
+          release: item.release_title.toLowerCase(),
```

Comment:
Incidentally, this unapproved change from the other PR is making the app crash:
```
react-dom.development.js:12056 Uncaught TypeError: Cannot read properties of undefined (reading 'toLowerCase')
    at RemoteImageSelector.tsx:232:39
    at Array.map (<anonymous>)
    at RemoteImageSelector (RemoteImageSelector.tsx:166:6)
    at renderWithHooks (react-dom.development.js:16305:18)
    at updateFunctionComponent (react-dom.development.js:19588:20)
    at beginWork (react-dom.development.js:21601:16)
    at beginWork$1 (react-dom.development.js:27426:14)
    at performUnitOfWork (react-dom.development.js:26557:12)
    at workLoopSync (react-dom.development.js:26466:5)
    at renderRootSync (react-dom.development.js:26434:7)
```

Reply:
Huh, this shouldn't have ended up in this branch, I'll reset the head to be without the commit about release_title.